content/get-started/01_get-started.en.md

```diff
@@ -13,25 +29,122 @@ If you want to install Cerebral into an existing project please...
 If you want an introduction you can keep reading and go through a small demo project. Just make sure you fulfill these requirements:
 
 * You have installed [Node](https://nodejs.org/en/) version 4 or later
-* You are okay with learning Cerebral using React and JSX. You can choose something else for your own project later though
+* You have installed [Git](https://git-scm.com/)
+* You are okay with learning Cerebral using React and JSX. You can choose a different Viewlayer (like [Inferno](http://infernojs.org//)) for your own project later though. 
 * You have a coffee or a beer, ready to learn something new
 
-Okay, lets get going. First you need to clone the tutorial repo.
 
-`git clone https://github.com/cerebral/cerebral-website-tutorial.git`
+### Installing the project
 
-Or use [Github Desktop](https://desktop.github.com/)
+Okay, lets get going. First you need to clone cerebral repo. So fire up your command line go to your drive and directory of choice and do:
 
-### Starting the project
-The tutorial runs on [react-create-app](https://facebook.github.io/react/blog/2016/07/22/create-apps-with-no-configuration.html) by Facebook. To run it, go to the project folder and install the dependencies, including Cerebral by running:
+`git clone https://github.com/cerebral/cerebral.git`
 
-`npm install`
+then do a  `cd cerebral/packages/cerebral-website-tutorial`
```

Comment:
hmm  so we want to push the regular tutorial users to use the whole monorepo setup? i was thinking that they should git clone just to get the files and then cd to the tutorial dir, do the npm install from there to get the stuff from npm and then start. remember, monorepo doesnt work on windows and we shouldnt leave this guys out.